views/partials/searchForm.hbs

```diff
@@ -0,0 +1,9 @@
+<form class="search" method="GET" action="/search">
```

Comment:
It looks like you don't actually use this template. Try to use it in `index.hbs` and `search.hbs`.

Reply:
I couldn't link to the partial in the main template, I wouldn't render, will have a try again